src/utils/mathUtils.ts

```diff
@@ -71,7 +71,7 @@ export function isNaN(n: any) {
  * Numbers represented as strings do not pass this function
  */
 export function isValidNumber(n: any) {
-  return typeof n === "number" && !isNaN(n) && isFinite(n);
+  return typeof n === "number" && n - n < 1;
```

Comment:
same effect, much faster. it's in the inner loop of rendering so i just want to make sure we're squeeze as much perf out as possible